app/models/twilio_redirection.rb

```diff
@@ -0,0 +1,31 @@
+class TwilioRedirection
+  def self.for(twilio_number)
+    if (location = ::Location.current.find_by(twilio_number: twilio_number))
+      Location.new(location.booking_location || location)
```

Comment:
Could we do this logic outside of the initializer arguments? It's a little dense here.